pandas/tests/series/test_datetime_values.py

```diff
@@ -682,6 +682,9 @@ def test_setitem_with_different_tz(self):
             [[pd.NaT], [[np.NaN, np.NaN, np.NaN]]],
             [["2019-12-31", "2019-12-29"], [[2020, 1, 2], [2019, 52, 7]]],
             [["2010-01-01", pd.NaT], [[2009, 53, 5], [np.NaN, np.NaN, np.NaN]]],
+            # see GH#36032
+            [["2016-01-08", "2016-01-04"], [[2016, 1, 5], [2016, 1, 1]]],
+            [["2016-01-07", "2016-01-01"], [[2016, 1, 4], [2015, 53, 5]]],
```

Comment:
is there another implementation somewhere (maybe stdlib calendar) we can use to compare against?

Reply:
i guess datetime.isocalendar would make sense

Reply:
https://github.com/python/cpython/blob/master/Modules/_datetimemodule.c#L3355

Reply:
@jbrockmendel I didn't want to change the current implementation too much, but I tested with `datetime.isocalendar` locally.

https://github.com/pandas-dev/pandas/pull/36050#discussion_r481532226

Reply:
Can you add in `tests.tslibs.test_ccalendar` (OK for follow-up):

```
from hypothesis import given, strategies as st

@given(st.datetimes(min_value=pd.Timestamp.min.to_pydatetime(), max_value=pd.Timestamp.max.to_pydatetime())
def test_isocalendar(dt):
    expected = dt.isocalendar()
    result = ccalendar.get_iso_calendar(dt.year, dt.month, dt.day)
    assert result == expected
```

Reply:
@jbrockmendel Unsure how to resolve the CI failures? should I just add `warn=False` in the `to_pydatetime` call?
https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=41452&view=ms.vss-test-web.build-test-results-tab&runId=2135520&resultId=100016&paneView=debug